docs/full_guide.md

```diff
@@ -222,6 +222,34 @@ commit-msg:
 
 When you try to commit `git commit -m "haha bad commit text"` script `template_checker` will be executed. Since commit text doesn't match the described pattern the commit process will be interrupted.
 
+## Bash script example with Commitlint
+
+Let's create a bash script to check conventional commit status `.lefthook/commit-msg/commitlint.sh`:
+
+```bash
+INPUT_FILE=$1
+START_LINE=`head -n1 $INPUT_FILE`
+ERROR=`echo $START_LINE | npx commitlint --color`
+if ! [[ $? == 0 ]]; then
+  echo "${ERROR}"
+  exit 1
+fi
```

Comment:
A separate shell script is not needed anymore after
https://github.com/evilmartians/lefthook/commit/8defa7e2a3bd4385d3a296056c232c28183328dc

Reply:
Can you please elaborate? I don't quite follow...